Bio/PopGen/GenePop/Controller.py

```diff
@@ -122,7 +122,7 @@ def _hw_func(stream, is_locus, has_fisher=False):
             # loci might mean pop if hook="Locus "
             loci = {}
             for entry in table:
-                if len(entry) < 3:
+                if len(entry) < 4:
```

Comment:
This looks like it was originally intended for pul request #583 (nothing to do with the Entrez pull request)